clippy_lints/src/methods/map_clone.rs

```diff
@@ -119,6 +122,9 @@ fn handle_path(
             && let args = args.as_slice()
             && let Some(ty) = args.iter().find_map(|generic_arg| generic_arg.as_type())
             && ty.is_ref()
```

Comment:
This line can now be removed, since the new `let ty::Ref` check also verifies this